core/src/display_object/edit_text.rs

```diff
@@ -850,6 +855,101 @@ impl<'gc> EditText<'gc> {
         }
     }
 
+    /// How many lines the text can be scrolled down
+    pub fn maxscroll(self) -> usize {
+        let edit_text = self.0.read();
+
+        // non-word-wrapped text can't be scrolled
```

Comment:
I see counterexamples, like:
```
tf.text = "1\n2\n3\n4\n5\n6\n7\n"; // only 4 lines fit
tf.multiline = true;
tf.wordWrap = false;
trace(tf.maxscroll); // 4
```

Reply:
hah, I failed to test anything with manual line breaks! Good spot!